docs/src/main/paradox/docs/delta/api/current/iam-realms-api.md

```diff
@@ -80,7 +80,7 @@ Response
  :   @@snip [realm-deleted-ref.json](assets/realms/realm-deleted-ref.json)
 
 
-## List realms
+## List
```

Comment:
For the listings, please take a look to projects listings (formatting and sort field)